careers/base/templates/base/header.html

```diff
@@ -20,7 +20,7 @@
         </nav>
         <h2 class="masthead-logo">
           <a href="{{ url('careers.home') }}">
-            <img class="ga-nav-home-logo" src="{{ static('img/header-mozilla-stone.png') }}" alt="mozilla">
+            Careers
```

Comment:
I think we should put "Mozilla" in front of this, as just the name "Careers" is a little vague. Thoughts?